help/scim.md

```diff
@@ -158,6 +157,47 @@ Zulip's SCIM integration has the following limitations:
 
     Once SCIM has been configured, consider also [configuring SAML](/help/saml-authentication).
 
+## Synchronizing group membership with SCIM
+
+Zulip also supports syncing of users' [groups](/help/user-groups) via SCIM.
+
+{start_tabs}
+
+{tab|okta}
+
+1. Follow the instructions [above](#configure-scim) to configure SCIM.
+
+1. Open the **Application** you set up above for the Zulip SCIM integration, and
+   go to the **Push groups** tab. This menu allows you to choose the Okta groups
+   which should be synchronized with Zulip's user groups.
+
+1. Make sure you're familiar with the below rules governing SCIM group sync behavior:
+
+    * When you set up an Okta group in the **Push groups** tab, the SCIM
+      integration will create a user group in Zulip with the matching name and
+      user memberships.
+    * When you add or remove users from the group in Okta, these changes will
+      be immediately be reflected in group memberships in Zulip.
+    * You can only enable synchronization of groups which do not yet exist in Zulip.
+      If you push a group whose name matches an existing Zulip group, the request
+      will fail.
+    * SCIM `DELETE` requests are not supported for groups. This means that if you choose
+      to **Unlink** a group in Okta from the Zulip SCIM integration, you **must** select
+      **Leave the group in the target app**. The **Delete the group in the target app**
+      option is not supported.
```

Comment:
this is the related practical advice to users due to merging without `DELETE` support